src/parser/core/modules/Gauge/CounterGauge.ts

```diff
@@ -48,11 +44,11 @@ export interface CounterGaugeOptions extends AbstractGaugeOptions {
 	deterministic?: boolean
 }
 
-export interface CounterChartOptions {
-	/** Label to display on the data set. */
-	label: string
-	/** Color to draw the data set in. Defaults to grey. */
-	color?: string | Color
+type CounterGraphOptions =
+	&Omit<ResourceGraphOptions, ''> // Not currently omitting any options, but making easier to do so in the future
```

Comment:
Any reason why there's an & here?

It might be cleaner to have a base `GraphOptions` interface which `ResourceGraphOptions` and `CounterGraphOptions` (and others) extend.

Reply:
The & I think is leftover from a previous iteration. Removed.

The trouble with starting from a base GraphOptions and adding to it elsewhere is that the graphing code needs to have access to all of the options, regardless of which kind of graph it is to know how to display it, but we don't want some of the gauge types making use of some of the options since they either don't make sense or don't interact well with the others. Ack suggested enforcing that by narrowing the options list, so that's what I did here.